astropy/wcs/tests/test_wtbarr.py

```diff
@@ -0,0 +1,66 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import pytest
+
+import numpy as np
+
+from astropy import wcs
+
+from . helper import SimModelTAB
```

Comment:
Perhaps this import is no longer needed since you are now using the fixture?

Reply:
Yeah. I got distracted and forgot to remove the line.